kcidb/db/__init__.py

```diff
@@ -89,6 +87,16 @@ def is_initialized(self):
         """
         return self.driver.is_initialized()
 
+    def reset(self):
+        """
+        Recreates a database driver instance on request
+        using existing database string
```

Comment:
This basically describes the implementation and is not very useful for the user. We need more of a conceptual description. Something like "Reset the client's database connection, updating its knowledge of the database schema."

Reply:
Please take care of this request as well, in the corresponding commit you're adding the function.